Graphics/Vulkan/include/NovelRT/Graphics/Vulkan/Utilities/BufferUsageKind.hpp

```diff
@@ -3,13 +3,15 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
 // for more information.
 
-#include <cstdint>
-#include <vulkan/vulkan.h>
 #include <NovelRT/Graphics/GraphicsBufferKind.hpp>
 #include <NovelRT/Graphics/GraphicsResourceAccess.hpp>
 #include <NovelRT/Graphics/GraphicsTextureKind.hpp>
-#include <NovelRT/Utilities/Misc.h> // we need this for the bitwise logic
 #include <NovelRT/Graphics/Vulkan/Utilities/Vma.hpp>
+#include <NovelRT/Utilities/Operators.hpp>
+
+#include <cstdint>
+
```

Comment:
nit - random new lines

Reply:
This isn't random - it's separating the includes based on where they're from. Ours, then stdlib, then thirdparty.